modules/ROOT/pages/building-soap-proxy.adoc

```diff
@@ -81,6 +81,24 @@ You can resolve this issue by adding the appropriate imports in the WSDL file. F
 </types>
 ----
 
+== Retrieving a WSDL Using Basic Authentication Credentials
+
+You can retrieve a WSDL, hosted on a remote site, protected with a basic authentication by using the SOAP proxy mechanism. You configure the `basic.auth.user` and `basic.auth.password` application properties to retrieve the WSDL. These properties are set in the `basicauth.properties` config file and are passed to the validator module:
+
+[source,xml,linenums]
+----
+<soap-request-validator:config name="validator-config" wsdlLocation="http://myremoteurl:9092/service?wsdl" serviceName="myServiceName"
+                                   servicePort="myServicePort" schemaValidationEnabled="true"
+                                   basicAuthUser="${basic.auth.user}" basicAuthPassword="${basic.auth.password}"/>
+----
+
+If no authentication values are provided for the properties, the WSDL is retrieved without the basic authentication.
```

Comment:
Does "without the basic authentication" mean that the WSDL is not authenticated but still retrieved? 
```suggestion
If you do not provide authentication values for the properties, the WSDL is retrieved unauthenticated.
```
What does that mean? 
This topic is about retrieving the WSDL by using authentication, so why are we mentioning NOT providing properties and NOT using authentication?
And how does that work, to retrieve the WSDL unauthenticated?

Reply:
Done! Thanks!